src/Pagination/utils/ContainerUtils.ts

```diff
@@ -49,9 +51,30 @@ export const hideLoader = (targetListView: DataSourceHelperListView) => {
     targetListView.domNode.classList.remove("widget-pagination-loading");
 };
 
-export const mxTranslation = (namespace: string, key: string, replacements: any[]) => {
-    const templateString = mx.session.getConfig(`uiconfig.translations.${namespace}.${key}`)
-        || (window.mx.session.getConfig("uiconfig.translations") as any)[`${namespace}.${key}`]
-        || "[No translation]";
-    return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+export const mxTranslation = (namespace: string, key: string, replacements: any[], lookAtWindow: boolean, defaultTemplateValue: string) => {
+    if (!lookAtWindow) {
+        const templateString = mx.session.getConfig(`uiconfig.translations.${namespace}.${key}`)
+            || (window.mx.session.getConfig("uiconfig.translations") as any)[`${namespace}.${key}`]
+            || "[No translation]";
+        return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+    } else if (window.__widgets_translations) {
+        const templateString = window.__widgets_translations[`${namespace}.${key}`]
+            || "[No translation]";
+        return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
+    } else {
+        return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), defaultTemplateValue);
+    }
```

Comment:
In the if statements set the `templateString`, and a use single return.
``` ts
return replacements.reduce((substituteMessage, value, index) => substituteMessage.split("{" + (index + 1) + "}").join(value), templateString);
```

Reply:
Done